components/Buttons/AddSetButton.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styled from "styled-components";
+import {Icon} from "@iconify/react";
+
+export const AddSetButton = ({onClick}) => {
+  return (
+    <>
+      <AddButton onClick={onClick}>
+        <Icon
+          icon="material-symbols:add"
+          width="25"
+          height="25"
+          title="Workout"
+          color="#735cdd"
+        />
+        Add Set
+      </AddButton>
+    </>
+  );
+};
```

Comment:
It's a lot of overhead to create a separate component for every button.
It would be more efficient to have a single `IconButton` component, e.g. with a property `variant` and possible values `add-set`, `delete-set` etc.. Depending on the value of this property you could render the matching icon.
You would avoid a lot of duplicate code.